index.py

```diff
@@ -580,9 +601,26 @@ def replace_insane(char):
             if type(err_i) == bytes:
                 err += err_i.decode(sys.stdout.encoding)
 
-        if len(err):
-            error(f"FFmpeg: {err}")
-
+        debug(f"FFmpeg complete log:\n{ff_logtext}\n")
+
+        # remove harmless warnings
+        err = err.split('\n')
+        for ignore in [
+            "    Last message repeated ",
+            "Found duplicated MOOV Atom. Skipped it",
+            "Found unknown-length element with ID 0x18538067 at pos."  # segment header
```

Comment:
alright, yeah FFmpeg will probably recover nicely from all similar issues :p 
I just figured it would be a good idea to be specific, since 0x18538067 is the key for video segment headers